core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -789,8 +790,19 @@ private void loadMetadata(LockedInodePath inodePath, LoadMetadataContext context
               LOG.debug("Failed to loadMetadata because file is not in ufs:"
                       + " inodePath={}, options={}.",
                   childURI, loadMetadataContext, e);
+            } catch (BlockInfoException | FileAlreadyCompletedException
+                | FileDoesNotExistException | InvalidFileSizeException
+                | IOException e) {
+              LOG.debug("Failed to loadMetadata because the ufs file or directory"
+                      + " is {}, options={}.",
```

Comment:
@yuzhu @ljl1988com Maybe not in this PR, but I think we should re-think on what exactly are we logging in DEBUG level. Logging too many lines may generate too much extra information. However we do want to be able to see the error trace sometime. 

Ideally if a file/dir succeeds to load, I only want to see 1 line max at DEBUG level, and 0 lines at INFO. If something fails to load, currently we expect at least 1 warning (and short) at INFO/WARN level, and at DEBUG level we want the stacktrace (maybe the status and options too, but they can also take many lines).

Sometime before I turn `InodeSyncStream` to DEBUG, just to collect how long each operation takes in `LOG.debug("synced {} paths ({} success, {} failed) in {} ms on {}",...)`. Maybe if you want to print much more details in InodeSyncStream, they should be in TRACE level?

Reply:
Also, there are try-catch blocks in callers of `loadMetadata()`, we'd like to make sure that if an exception is thrown, it's caught once and logged once. Instead of caught-print-rethrown-caught-print-rethrown.